code/modules/trade/datums/trade_stations_presets/asterstradecapital.dm

```diff
@@ -0,0 +1,119 @@
+/datum/trade_station/station_asterstradecapital
+    name_pool = list("FTS 'Solaris'" = "Free Trade Station 'Solaris', they sending message \"Hello, this is the Trade Station 'Solaris'. We have all of the bests products on sale at Hansa! You couldn't get even better prices!.\ Everything for sale here, don't be afraid to come aboard and check our wares!\".") // This is, going tobe the main trading station. Has a grand part of the not so big things, and a fuckload of old crates of the shuttle.
+    start_discovered = TRUE
+    spawn_always = TRUE
+    markup = 0.6
+
+    forced_overmap_zone = list(
+        list(24, 26),
+        list(24, 30)
+    )
+/*
+
+TODO: FIX NAMES SO ITS " Crowbar" and not "crowbar", add other designs.
+
+*/
+    assortiment = list(
+        "Disk Designs" = list(
+            /obj/item/weapon/computer_hardware/hard_drive/portable/design/tools = good_data("Asters Basic Tool Pack", list(1, 10)),
+            /obj/item/weapon/computer_hardware/hard_drive/portable/design/misc = good_data("Asters Miscellaneous Pack", list(1, 10)),,
+            /obj/item/weapon/computer_hardware/hard_drive/portable/design/robustcells = good_data("Asters Robustcells", list(1, 10)),,
+            /obj/item/weapon/computer_hardware/hard_drive/portable/design/devices = good_data("Asters Devices and Instruments", list(1, 10)),,
+        ),
+
+        "Tools and Equipment" = list(
+            /obj/spawner/cloth/holster,
+            /obj/item/weapon/storage/pouch/small_generic,
+            /obj/item/weapon/storage/pouch/ammo,
+            /obj/item/weapon/storage/pouch/gun_part,
+            /obj/item/clothing/suit/storage/hazardvest,
+            /obj/item/weapon/storage/belt/utility = good_data("Empty Toolbelt", list(5, 20)),
+            /obj/item/weapon/storage/belt/utility/full = good_data("Basic Tools and Toolbelt", list(2,8)),
+            /obj/item/device/lighting/toggleable/flashlight,
+            /obj/item/device/lighting/toggleable/flashlight/heavy,
+            /obj/item/weapon/tool/omnitool = good_data("Asters \"Munchkin 5000\"", list(2, 8)),
+            /obj/item/weapon/tool/crowbar,
+            /obj/item/weapon/tool/screwdriver,
+            /obj/item/weapon/tool/wrench,
+            /obj/item/weapon/tool/shovel,
+            /obj/item/weapon/tool/weldingtool,
+            /obj/item/weapon/tool/screwdriver,
+            /obj/item/weapon/tool/tape_roll,
+            /obj/item/weapon/tool/wirecutters,
+            /obj/spawner/tool_upgrade = good_data("Random Toolmod", list(0, 50)), // ditto of an actual crate. Though, could be switched into a proper category to make people able to buy specific ones already or move it into another trade station/ship.
```

Comment:
This won't work out good because there's no fucking price for it like the disk spawner

Reply:
It, actually has a price. Since I had to test why some prices were zero (they lacked ap roper tag, and I still have to add these to half of items I added to this trade station and others.

The random toolmod seems to be priced on 500 credits (and some decimals) for some reason. Even when I tried to go around and find it, I had no luck. Probably I just missed it, also a something similar happens with RIGs, which seems to be at 686.000~

I guess these are inheriting the price from something. Adding a tag fixed it, anyway. I don't know if adding prices to spawners is correct, though.